.github/workflows/unittests.yml

```
@@ -12,9 +12,9 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: windows-latest, r: "4.0.5"}
-          - {os: macOS-latest,   r: "4.0.5"}
-          - {os: ubuntu-20.04,   r: "4.0.5"}
+          - {os: windows-latest, r: "4.1"}
+          - {os: macOS-latest,   r: "4.1"}
+          - {os: ubuntu-20.04,   r: "4.1"}
```

This is probably fine, but it rubs me the wrong way
```suggestion
          - {os: windows-latest, r: "4.1.0"}
          - {os: macOS-latest,   r: "4.1.0"}
          - {os: ubuntu-20.04,   r: "4.1.0"}
```